config/default.schema.json

```diff
@@ -1215,6 +1215,41 @@
 					"description": "Number of simultaneously synchronized students, teachers and classes"
 				}
 			}
+		},
+		"FEATURE_TLDRAW_CALLBACK_URL": {
+			"type": "string",
+			"default": "",
+			"description": "callback URL for tldraw communication "
+		},
+		"FEATURE_TLDRAW_CALLBACK_TIMEOUT": {
+			"type": "number",
+			"default": 5000,
+			"description": "callback timeout for tldraw communication "
+		},
+		"FEATURE_TLDRAW_CALLBACK_OBJ": {
+			"type": "string",
+			"default": "",
+			"description": "callback object for tldraw communication "
+		},
+		"FEATURE_TLDRAW_CALLBACK_DEBOUNCE_WAIT": {
+			"type": "number",
+			"default": 2000,
+			"description": "time for waiting between calls for tldraw"
+		},
+		"FEATURE_TLDRAW_CALLBACK_DEBOUNCE_MAX_WAIT": {
+			"type": "number",
+			"default": 10000,
+			"description": "Max time for waiting between calls for tldraw"
+		},
+		"FEATURE_TLDRAW_PING_TIMEOUT": {
+			"type": "number",
+			"default": 30000,
+			"description": "Max time for waiting between calls for tldraw"
+		},
+		"FEATURE_TLDRAW_GC_ENABLED": {
+			"type": "boolean",
+			"default": true,
+			"description": "If tldraw garbage collector should be enabled"
```

Comment:
I think this `FEATURE_` prefix (and `FEATURE_..._ENABLED` format in general) is now reserved for the feature flags, e.g. if Tldraw would be possible to be turned on/off (if it could be treated as a feature), the feature flag for it should be named `FEATURE_TLDRAW_ENABLED`, but a specific feature params should be named with just the feature name as a prefix so in this case I would name these envs just
```
TLDRAW_CALLBACK_URL
TLDRAW_CALLBACK_TIMEOUT
...
```
but would also like to hear @virgilchiriac opinion on that.